internal/buffer/prefixer_test.go

```diff
@@ -0,0 +1,43 @@
+package buffer_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/scoir/nats-websocket-gw/internal/buffer"
+)
+
+func TestWebsocketPrefixing(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    string
+		expected string
+	}{
+		{name: "non-sub command is untouched", input: "PING", expected: "PING"},
+		{name: "sub command gets prefixed", input: "SUB abc.xyz 2", expected: "SUB prefixed.abc.xyz 2"},
+		{name: "sub command with multiple spaces gets prefixed", input: "SUB      abc.xyz 2", expected: "SUB prefixed.     abc.xyz 2"},
```

Comment:
I don't like the outcome of this test, but it seems to be working like it was before my changes